tests/test_datasets/test_formatting.py

```diff
@@ -46,7 +48,13 @@ def test_transform(self):
         self.assertEqual(results['data_samples'].ori_shape,
                          results['data_samples'].gt_sem_seg.shape)
         results = copy.deepcopy(self.results)
+        # test dataset shape is not 2D
         results['gt_seg_map'] = np.random.rand(3, 300, 400)
+        with pytest.warns(UserWarning):
+            warnings.warn('Please pay attention your ground truth '
+                          'segmentation map, usually the segentation '
+                          'map is 2D, but got '
+                          f'{results["gt_seg_map"].shape}')
         results = transform(results)
```

Comment:
Do you mean assert warning like this?
```suggestion
        msg = 'Please pay attention your ground truth '
                   'segmentation map, usually the segentation '
                   'map is 2D, but got '
                   f'{results["gt_seg_map"].shape}')
        with pytest.warns(UserWarning, match=msg):
            results = transform(results)
```